components/auth/Verify.tsx

```diff
@@ -0,0 +1,141 @@
+import React, { ReactElement } from "react";
+import { useRouter } from "next/router";
+import { Formik } from "formik";
+import { TextInput, Label, Alert, Description } from "@components/forms";
+import { useTranslation } from "next-i18next";
+import * as Yup from "yup";
+import Link from "next/link";
+import { ErrorStatus } from "@components/forms/Alert/Alert";
+import { useAuthErrors } from "@lib/hooks/auth/useAuthErrors";
+import { Button } from "@components/globals";
+import { useVerify } from "@lib/hooks/auth";
+
+// TODO:
```

Comment:
Let's remove this

Reply:
Got it, will replace <Confirmation> with <Verification>